provisioner.go

```diff
@@ -185,8 +185,13 @@ func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher
 
 // provisionPackage provisions a package
 func (p *defaultProvisioner) provisionPackage(ctx context.Context, d Dispatcher, paths Paths, s *ProvisionStatusPackage, m mover, name, version, pathUnzipSrc, pathDirectory string, finish func() error) (err error) {
+	_, err = os.Stat(pathDirectory)
```

Comment:
If the dir is not created but the zip is downloaded, then we can only unzip. Because we store the electron zip file but not electron dir on the git.